ark/utils/spatial_analysis_utils_test.py

```diff
@@ -99,6 +99,18 @@ def make_example_data_closenum():
     return all_data, dist_mat
 
 
+def make_example_neighbor_mat():
+    col_names = {0: 'feature_1', 1: 'feature_2'}
+    neighbor_counts = pd.DataFrame(np.zeros((60, 5)))
+
+    neighbor_counts.iloc[0:15, 0:2] = np.random.randint(low=0, high=10, size=(15, 2))
```

Comment:
Can we add the previous function to test_utils and subset it here to keep the columns we need?

Reply:
To clarify, you mean move `make_example_data_closenum` to `test_utils`? And then we call it to create `neighbor_counts`?

Reply:
Can you move `make_neighborhood_matrix` from spatial_analysis_test to test_utils?

Reply:
Got it.